runtime/src/contracts_config.rs

```diff
@@ -0,0 +1,97 @@
+use crate::{
+    Balance, Balances, BalancesCall, Perbill, Runtime, RuntimeCall, RuntimeEvent,
+    RuntimeHoldReason, Timestamp, MILLIUNIT, UNIT,
+};
+use frame_support::{
+    parameter_types,
+    traits::{ConstBool, ConstU32, Randomness},
+};
+use frame_system::pallet_prelude::BlockNumberFor;
+
+pub enum AllowBalancesCall {}
+
+impl frame_support::traits::Contains<RuntimeCall> for AllowBalancesCall {
+    fn contains(call: &RuntimeCall) -> bool {
+        matches!(
+            call,
+            RuntimeCall::Balances(BalancesCall::transfer_allow_death { .. })
+        )
+    }
+}
+
+const fn deposit(items: u32, bytes: u32) -> Balance {
+    (items as Balance * UNIT + (bytes as Balance) * (5 * MILLIUNIT / 100)) / 10
+}
+
+fn schedule<T: pallet_contracts::Config>() -> pallet_contracts::Schedule<T> {
+    pallet_contracts::Schedule {
+        limits: pallet_contracts::Limits {
+            runtime_memory: 1024 * 1024 * 1024,
+            ..Default::default()
+        },
+        ..Default::default()
+    }
+}
+
+// randomness-collective-flip is insecure. Provide dummy randomness as placeholder for the deprecated trait.
+//https://github.com/paritytech/polkadot-sdk/blob/9bf1a5e23884921498b381728bfddaae93f83744/substrate/frame/contracts/mock-network/src/parachain/contracts_config.rs#L45
+pub struct DummyRandomness<T: pallet_contracts::Config>(sp_std::marker::PhantomData<T>);
+
+impl<T: pallet_contracts::Config> Randomness<T::Hash, BlockNumberFor<T>> for DummyRandomness<T> {
+    fn random(_subject: &[u8]) -> (T::Hash, BlockNumberFor<T>) {
+        (Default::default(), Default::default())
+    }
+}
+
+parameter_types! {
+    pub const DepositPerItem: Balance = deposit(1, 0);
+    pub const DepositPerByte: Balance = deposit(0, 1);
+    pub Schedule: pallet_contracts::Schedule<Runtime> = schedule::<Runtime>();
+    pub const DefaultDepositLimit: Balance = deposit(1024, 1024 * 1024);
+    pub const CodeHashLockupDepositPercent: Perbill = Perbill::from_percent(0);
+}
+
+impl pallet_contracts::Config for Runtime {
+    type Time = Timestamp;
+    type Randomness = DummyRandomness<Self>;
+    type Currency = Balances;
+    type RuntimeEvent = RuntimeEvent;
+    type RuntimeCall = RuntimeCall;
+
+    /// The safest default is to allow no calls at all.
+    ///
+    /// Runtimes should whitelist dispatchables that are allowed to be called from contracts
+    /// and make sure they are stable. Dispatchables exposed to contracts are not allowed to
+    /// change because that would break already deployed contracts. The `RuntimeCall` structure
+    /// itself is not allowed to change the indices of existing pallets, too.
+    type CallFilter = AllowBalancesCall;
+    type DepositPerItem = DepositPerItem;
+    type DepositPerByte = DepositPerByte;
+    type CallStack = [pallet_contracts::Frame<Self>; 23];
```

Comment:
Note: probably doesnt matter for now, but Trappist has this configured as 5: https://github.com/paritytech/trappist/blob/a07b7e84224c53d16dbd1d5e3fb8797f33d12723/runtime/trappist/src/contracts.rs#L58